src/test/java/com/microsoft/sqlserver/jdbc/datatypes/DataTypesTest.java

```diff
@@ -1779,4 +1785,85 @@ public void testDateTimeInsertUpdate() throws Exception {
             }
         }
     }
+
+    /**
+     * Test example from https://github.com/microsoft/mssql-jdbc/issues/1088
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testGetLocalDateTimePriorGregorian() throws Exception {
+        try (Connection conn = getConnection(); Statement st = conn.createStatement();) {
+            String ldtTable = AbstractSQLGenerator.escapeIdentifier(RandomUtil.getIdentifier("ldtTable"));
+            TestUtils.dropTableIfExists(ldtTable, st);
+            // test data (The Battle of Hastings)
+            LocalDateTime ldtExpected = LocalDateTime.of(1066, 10, 14, 0, 0);
+            st.execute("CREATE TABLE " + ldtTable + " (id int PRIMARY KEY, dt datetime2)");
+            PreparedStatement ps = conn.prepareStatement("INSERT INTO " + ldtTable + " (id, dt) VALUES (1, ?)");

```

Comment:
Move the PreparedStatement  to try block

Reply:
great catch